_irc/index.md

```diff
@@ -260,6 +266,31 @@ covers the `label` tag, and how clients should send and will receive
 labels.
 
 
+## Listing Users
+
+IRCv3 specifications extend traditional queries to carry more information about
+other users in a given channel.
+
+The [`multi-prefix` spec]({{site.baseurl}}/specs/extensions/multi-prefix.html)
+details the exact messages these changes apply to and how exactly it's used.
```

Comment:
This sentence was originally after the next one (which acted as a kind of introduction). Now this reads a bit weird. Maybe we could rejigger the text a bit?